### ISSUE TITLE
Is cheatengine virus?

### ISSUE THREAD

**Issue by qw854** [cheat-engine/cheat-engine#2286]
I searched it on virus but 5 sources says "It's not virus, it's marked as virus by some antiviruses because it injects code to a running process". But to be sure i want to be ask official CheatEngine Community, it its not virus can someone send a video that shows downloading latest CheatEngine and opening it?

**Comment by cheat-engine** [comment]
https://www.youtube.com/watch?v=v7QsS5qp0og

**Comment by avboy1337** [comment]
build it self is also a methord to make sure safe to use.

**Comment by guvra** [comment]
The installer is, the application is not.

**Comment by qw854** [comment]
> The installer is, the application is not.

**_yeah, the installer contains RATs, viruses, trojan._**

**Comment by cheat-engine** [comment]
nope. The installer only contains only verified applications .  The 'virus' in this case is the actual Cheat Engine installer part

**Comment by zMohawk** [comment]
It also finds Win32\Vigua.a trojan, quite strange.
